statistics/histogram.go

```diff
@@ -742,6 +742,18 @@ func (c *Column) GetColumnRowCount(sc *stmtctx.StatementContext, ranges []*range
 			}
 			continue
 		}
+		rangeVals := enumRangeValues(rg.LowVal[0], rg.HighVal[0], rg.LowExclude, rg.HighExclude)
+		// The small range case.
```

Comment:
Does it seem that we don't check the condition `the bucket is large` now?
Or is there no necessary to check it?

Reply:
Which check do you mean?